Added loader to patientview

elephantgloves opened the pull request:
Please check loader utility.

elephantgloves commented:
Fixed linter messages. Please check again.

tuttieee commented on src/connects/PatientView.react.js:
```diff
@@ -56,6 +56,7 @@ const mapStateToProps = (state) => {
     records: getActiveRecords(state),
     selectedActiveRecordIndex: getSelectedActiveRecordIndex(state),
     isNew: !patient.hasOwnProperty('_rev'),
+    // isFetching: true,   // debug mode
```
Please remove these unnecessary lines

elephantgloves replied:
Removed unnecessary lines above - please check

elephantgloves commented:
1) Fixed linter errors
2) Rewrote asha:sum function to use spread operator, reduce, arg. length checking instead of exception catching

確認
3) What to do about class name? (mul uses asha:math:mul whereas sum currently uses asha:sum)
